packages/cli/src/utils.ts

```diff
@@ -20,20 +13,50 @@ export const runCommand: RunCommandFunc = async (command, options) => {
   return execa.execaCommandSync(command, options)
 }
 
-//TODO testing
-export const validateToken = (token?: string): string | null => {
+type GetPersonalAccessTokenFunc = (params: {
+  token?: string
+  dotEnvPath: string | undefined
+}) => { error?: false; token: string } | { error: true; message: string }
+
+export const getPersonalAccessToken: GetPersonalAccessTokenFunc = ({
+  token,
+  dotEnvPath,
+}) => {
   if (typeof token !== 'undefined' && token !== '') {
-    return token
+    return {
+      token,
+    }
   }
 
-  if (
-    typeof process.env.STORYBLOK_PERSONAL_ACCESS_TOKEN !== 'undefined' &&
-    process.env.STORYBLOK_PERSONAL_ACCESS_TOKEN !== ''
-  ) {
-    return process.env.STORYBLOK_PERSONAL_ACCESS_TOKEN
+  const pathsToLoad =
+    typeof dotEnvPath === 'string' ? [dotEnvPath] : ['.env', '.env.local']
+
+  if (pathsToLoad.every((path: string) => !existsSync(path))) {
```

Comment:
what happens if pathsToLoad is the default therefore ['.env', '.env.local'] but the project only had .env or .env.local and not both

Reply:
what this line does is "return an error if none of them exists"

So if only one of them exists, it still goes on.

Reply:
done! https://github.com/storyblok/field-plugin/pull/109/commits/d3122daf8a8f5262161addab9b3d768e71f364e1